resources/config.rb

```diff
@@ -0,0 +1,66 @@
+unified_mode true
+
+property :fpm_service, String,
+  default: lazy { php_fpm_service },
+  description: 'Name of the PHP-FPM service to manage.
+Defaults to platform specific names, see libraries/helpers.rb'
+
+property :fpm_ini_control, [true, false],
+  default: lazy { php_fpm_ini_control },
+  description: 'Whether to manage the PHP-FPM configuration file or not.
+Defaults to false'
+
+property :fpm_conf_dir, String,
+  default: lazy { php_fpm_conf_dir },
+  description: 'Directory path of the PHP-FPM configuration file.
+Defaults to platform specific paths, see libraries/helpers.rb'
+
+property :ini_template, String,
+  default: lazy { php_ini_template },
```

Comment:
Remove `php_ini_template` method
```suggestion
  default: 'php.ini.erb',
```